airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -432,7 +432,8 @@ def download_data(self, url: str, chunk_size: int = 1024) -> tuple[str, str]:
         # set filepath for binary data from response
         tmp_file = os.path.realpath(os.path.basename(url))
         with closing(self._send_http_request("GET", f"{url}/results", stream=True)) as response, open(tmp_file, "wb") as data_file:
-            response_encoding = response.encoding or self.encoding
+            self.logger.info(f"Logging encoding values \n actual header: {response.headers} \nresponse.encoding : {response.encoding} \n response.apparent_encoding : {response.apparent_encoding} \n self.encoding : {self.encoding}")
+            response_encoding = self.encoding
```

Comment:
Is this related to the change we had to do with the encoding?

Reply:
Yes. This change was in a custom branch. So added it to main as we will release from main now